grafana/provisioning/alerting/fan_is_stopped_alert_rule.yaml

```diff
@@ -8,7 +8,7 @@ groups:
     interval: 60s
     rules:
       - uid: WIrA8dHVz
-        title: Fan is stopped
+        title: Fan has stopped
```

Comment:
This is a tricky one. We say "Disk is unhealthy" in another alert, where unhealthy is the state of the disk. In this case  stopped is the state of the fan. I also see how "Fan has stopped" makes sense, but it also seems inconsistent.

Reply:
I also see that I wrote has stopped below, so I guess it was inconsistent with that as well. I don't know.

Reply:
All good, my friend. This is why you have a technical writer. 😄 
I chose to go with `Fan isn't running` which is I think the point of urgency, right?